src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -663,7 +663,7 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
     <comment>{0} is the number of errors produced and {1} is the number of warnings.</comment>
   </data>
   <data name="VerifyCommand_Success" xml:space="preserve">
-    <value>Successfully verified package(s).</value>
+    <value>Successfully verified package.</value>
```

Comment:
Change to The package '{0}' was successfully verified.

Reply:
Why make this change?  The verify command can be called on more than one package.

    nuget.exe verify -Signatures *.nupkg